R/dsl-distributions.R

```diff
@@ -232,6 +232,8 @@ distr_poisson <- distribution(
 distr_truncated_normal <- distribution(
   name = "TruncatedNormal",
   density = function(x, mean, sd, min, max) {
+    dnorm(x, mean, sd, log = TRUE) - 
```

Comment:
Does this need checks to make sure we return -Inf outside of the domain?

Reply:
Good catch, I've fixed that